autoload/codefmt/formatterhelpers.vim

```diff
@@ -87,21 +87,32 @@ endfunction
 " the tool for range formatting and post a URL for that feature request above
 " code that calls it.
 "
-" @throws ShellError if the {cmd} system call fails
+" vararg0: if truthy, the syscall ignores errors. This can be helpful for
+" formatters that return nonzero results for reasons unrelated to formatting.
+"
+" vararg1: If set to a nonzero number N, skip the first N lines of the
+" formatter output. This can be used to trim always-present headers. 
+"
+" @throws ShellError if the {cmd} system call fails (and vararg0 is not set)
 " @throws WrongType
 function! codefmt#formatterhelpers#AttemptFakeRangeFormatting(
-    \ startline, endline, cmd) abort
+    \ startline, endline, cmd, ...) abort
   call maktaba#ensure#IsNumber(a:startline)
   call maktaba#ensure#IsNumber(a:endline)
 
+  let l:ignoreerrors = a:0 >= 1 ? a:1 : 0
```

Comment:
Might still want to explicitly verify types on these if they're passed.

Reply:
Done.